minimesos/src/main/java/com/containersol/minimesos/docker/DockerContainersUtil.java

```diff
@@ -121,6 +123,7 @@ public DockerContainersUtil kill() {
      * Removes all containers in the util object
      *
      * @param ignoreFailure - use <code>true</code> if you expect containers might be stopped by this time
+     * @return TODO(mwl)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1135)